### PR TITLE
feat: add help center and update help links

### LINKED ISSUE #30: Add In-Product Help
SUMMARY:
1. We would like to add in-product DETAILED HELP in THREE PLACES in the Console app. One is after the user clicks on the icon next to the "Get started with a ready template or upload your own SDL" line and the other for when the user clicks the "help" icon at the bottom of the left side bar (the "?")
2. We would also like to add TOOLTIPS in FOUR places in the app

IN-PRODUCT DETAILED HELP IN SIDE BAR
================================

- [ ] FIRST ITEM. Help after "Get started with a ready template or upload your own SDL"

Two changes requested here:
1. Add "what is SDL?" label/ icon. Clicking it should pop open a side bar as shown below
Screen Shot 2022-12-20 at 10 14 10 AM

2. Display the following content in the side bar in the format indicated in the design above (ignore the orange comment icon)
======
"SDL stands for “Stack Definition Language” and, as the name suggests, is a way for you (as a user or tenant) to specify what infrastructure stack you need for your application. The SDL file includes compute needs, locations, pricing and other things that help Akash determine the best provider(s) for you to run your application on.

To dig deeper into SDL and it's various sections, read this [documentation](https://docs.akash.network/readme/stack-definition-language) 

To see some samples SDL configurations, check out this [awesome akash repository](https://github.com/ovrclk/awesome-akash). Each directory in the awesome akash repository contains a sample "deploy.yaml" SDL file that deploys a specific application on to akash"!

======

 - [ ] SECOND ITEM: Clicking Help (?) on the left nav bar, should pop open the side bar with the following elements. The topy and hyperlinks are provided below (hyperlinks should open in new tabs)
Screen Shot 2022-12-20 at 10 18 28 AM

Copy for the above design:

"Documentation": "Review detailed documentation" (hyperlink: https://docs.akash.network/guides/deploy)
"Discussion Forum": "View current and past community discussions" (https://github.com/orgs/akash-network/discussions)
"Discord Server": "Join the Discord server and ask any questions" (https://discord.akash.network/)
"Github Repository": "Check out the Akash Network Open Source code bases and community repository" (https://github.com/akash-network)

 - [ ] THIRD ITEM:
Wallet Help (@aktdenis - can you please add link and screenshot to the figma designs for this? I know we worked on them together but I can't seem to locate it)
Two places we will want to guide people: 1) On the top right "Wallet Connected" action and 2) On the "Connect Wallet" button in the deployment configuration screen (once we implement it).
The help for this needs to be a detailed help in the right side nav bar. Text on the right side nav bar should be as follows:
==========
"Deploying workloads on Akash Network requires you to pay in [Akash Token ($AKT)](https://akash.network/token) using a [Keplr Wallet](https://www.keplr.app/).

To use Keplr wallet with Overclock Console, [install the Keplr browser plugin](https://www.keplr.app/download) for your browser.
There are two main ways to fund your Keplr Wallet with AKT: 
1. Purchase them from one of the exchanges listed below:

- [Kucoin](https://www.kucoin.com/)
- [Crypto.com](https://crypto.com/us)
- [Kraken](https://www.kraken.com/)
- [AscendEx](https://ascendex.com/)
- [Gate.io](https://www.gate.io/)
- [Bittrex](https://bittrex.com/)
- [DigiFinex](https://www.digifinex.com/)

2. If you have tokens of other Cosmos chains you can exchange them for AKT using the [Osmosis App](https://app.osmosis.zone/).

For a step-by-step instruction on all that, consult our documentation on topic [here](https://docs.akash.network/tokens-and-wallets)"

===========

TOOL TIP HELP
===========

- [ ] 1. Hover + tooltip after "Configure Services" here
Screen Shot 2022-12-20 at 10 16 43 AM

Text: "This is where you can specify various parameters (one per service) that make up your SDL file. The number and name of the services will vary depending on the application/ SDL"

- [ ] 2. Hover + tooltip help after "pricing" here:

text: "The value you specify here is the maximum you would like to pay for the compute resources you are requesting for this service. uakt allows you to specify an amount that is a fraction of 1AKT"

- [ ] 3. Hover+tooltip text after "Resources" 
Screen Shot 2022-12-20 at 10 16 58 AM
Text: "This is where you specify how much CPU, memory and storage you would like to lease from the provider, to host your application."

- [ ] 4. Deployment details screen 
   - Hover tooltip help next to "Update Deployment" with text "This will update a limited set of fields in an existing/ active deployment. Compute resources and placement criteria are not updatable."
   - Hover tooltip help next to "Re-Deploy" with text "This will create a whole new replica of this deployment. The existing deployment will not be touched."

### PR DESCRIPTION
**Pull request by CrystalDime**
Adds a help center feature and update various context help

fixes #30

### CODE REVIEW

**Comment by CrystalDime** [comment]
https://user-images.githubusercontent.com/45443045/229912608-619d6a38-f1c0-423f-8f97-bbf1b5d0f4e8.mp4

**Comment by anilmurty** [comment]
perfect, thanks @CrystalDime ! - tagging @jtary to review/ merge

File: web/src/components/HelpCenter/HelpCenterSDL.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          {
+            type: 'card',
+            title: 'View on Docs',
+            body: "To dig deeper into SDL and it's various sections, read this: http://docs.akash.network/readme/stack-definition-language",
+            link: 'http://docs.akash.network/readme/stack-definition-language',
+          },
+          {
+            type: 'card',
+            title: 'View on Github',
+            body: "To see some sample SDL configurations. Check out this awesome akash repository. Each directory in the awesome akash repository contains a sample 'deploy-yami' SDL file that deploys a specific application on to-akash!",
```
Comment (anilmurty) [inline web/src/components/HelpCenter/HelpCenterSDL.tsx]: "To see some sample SDL configurations. Check out..." s/be "o see some sample SDL configurations, check out..."

Comment (anilmurty) [reply web/src/components/HelpCenter/HelpCenterSDL.tsx]: Also 'deploy-yami' should be 'deploy.yaml'

File: web/src/components/HelpCenter/HelpCenterWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        contentData={[
+          {
+            type: 'contentBody',
+            contentTitle: 'Why Keplr Wallet?',
+            contentBody: "Deplying workloads on Akash Network requires you to pay in <strong> Akash Token ($AKT) </strong> using a <strong>Keplr Wallet </strong>"
+          },
+          {
+            type: 'card',
+            title: 'Download Extension',
+            body: "To use Keplr wallet with the Overclock Console, install the Keplr browser plugin for your browser",
```
Comment (anilmurty) [inline web/src/components/HelpCenter/HelpCenterWallet.tsx]: please change "the Overclock Console" to "Akash Console" (note: drop the "the" also)

File: web/src/components/HelpCenter/HelpCenterWallet.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    
+           contentBody: "Purchase them from one of the exchanges listed below: ",
+           body: "If you have tokens of other Cosmos chains you can exchange them for AKT using the Osmosis App",
+           title: "Osmosis App",
+           link: 'https://app.osmosis.zone/'
+          },
+          {
+            type: 'card',
+            title: 'Akash Docs/ Wallet Setup',
+            body: "For a step-by-step instruction on all that, consult our documentation on topic here",
```
Comment (anilmurty) [inline web/src/components/HelpCenter/HelpCenterWallet.tsx]: please remove "on topic"

**Comment by CrystalDime** [comment]
changed

**Review by jtary — APPROVED** [review]
LGTM

**Comment by anilmurty** [comment]
hey @CrystalDime - I'm starting a discord channel for Console OSS devs. This will be used to communicate new tasks, roadmap/ plans and also enable better communication between devs that contribute regularly. If you are interested in being a part of this send me your discord handle to anil [at] akash.network or share it here.

**Pull request merged**